cross/libcap/Makefile

```diff
@@ -0,0 +1,33 @@
+PKG_NAME = libcap
+PKG_VERS = 2.31
+PKG_EXT = tar.xz
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://mirrors.edge.kernel.org/pub/linux/libs/security/linux-privs/libcap2/
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+DEPENDS =
+
+HOMEPAGE = http://sites.google.com/site/fullycapable/
+COMMENT  = POSIX 1003.1e capabilities (library). Libcap implements the user-space interfaces to the POSIX 1003.1e capabilities available in Linux kernels. These capabilities are a partitioning of the all powerful root privilege into a set of distinct privileges. 
+LICENSE  = GPLv2
+
+GNU_CONFIGURE = 1
+CONFIGURE_TARGET = nop
+COMPILE_TARGET = libcap_compile_target
+INSTALL_TARGET = libcap_install_target
+
+ADDITIONAL_CFLAGS = -O2 -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64
+
+include ../../mk/spksrc.cross-cc.mk
+
+# force install in /lib rather than /lib64 for x86_64
+# and install lib only (cross compile bin files does not work)
+.PHONY: libcap_install_target
+libcap_install_target:
```

Comment:
I prefer without `_target` suffix, it is obviously a make target.

Reply:
And I was used to take the original with pkg_name as prefix, since there is not such an example in the wiki page (except for POST_STRIP -> extra_install).
I will use here your preference....